frontend/src/component/feature/FeatureStrategy/RecentlyUsedConstraints/useRecentlyUsedConstraints.ts

```diff
@@ -0,0 +1,53 @@
+import { useLocalStorageState } from 'hooks/useLocalStorageState';
+import type { IConstraint } from 'interfaces/strategy';
+
+export const areConstraintsEqual = (
```

Comment:
This might be temporary or permanent. Wanted to make that if user saves same thing twice in sequence, it does not duplicate, but since there is no id, it must check fields.